### PR TITLE
LPD-28891 Key is not automatically generated when writing new Specifications label

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-28891

### CODE REVIEW

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
thank you!
Remeber to add the entities created to the dataguard

**Comment by monaco-fabio** [comment]
ci:test:sf

File: modules/test/playwright/tests/commerce/commerce-product-options-web/commerceProductSpecifications.spec.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	}
+
+	const optionCategory =
+		await apiHelpers.headlessCommerceAdminCatalog.getOptionCategories();
+
+	for (let i = 0; i < optionCategory.totalCount; i++) {
+		apiHelpers.data.push({
+			id: optionCategory.items[i].id,
+			type: 'optionCategory',
+		});
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-product-options-web/commerceProductSpecifications.spec.ts:93]: this is a little to much, can you check the specification and option category names before put those into the dataguard?

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by monaco-fabio** [comment]
I'm closing this PR. Resent here: #5110

**Pull request closed without merging**